packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -666,49 +666,93 @@ class ScrollableBase extends Component {
 		}
 	}
 
+	notifyVoiceException = (e, type) => {
+		const param = {'voiceUi': {'exception': type}};
+		if (window.webOSVoiceReportActionResult) {
+			window.webOSVoiceReportActionResult(param);
```

Comment:
I don't think we need to define the `param` variable. It would be better to pass `{voiceUi: {exception: type}}` object directly as the first parameter of the `webOSVoiceReportActionResult` function.

Reply:
It seems more clean. I will fix it.